calcom/cal.com
# chore: Update GH action from crowdin to latest version

## What does this PR do?

- updates the github action from crowdin to the latest version

Fixes # [(issue)](https://github.com/calcom/cal.com/issues/11354)

## Type of change


## How should this be tested?


This PR requires a follow up PR to be merged: 
- re-open & merge https://github.com/calcom/cal.com/pull/10416
- observe the following commit by the crowdin GitHub action: It should _not_ overwrite the changes from the above PR

## Mandatory Tasks
